packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -298,13 +298,15 @@ class InProgressComponent extends React.Component<
           <IconWithName
             status={reg.registration?.status || SUBMISSION_STATUS.DRAFT}
             name={NameComponent}
+            isDuplicate={!!reg.registration?.duplicates}
```

Comment:
it should be safer to check length of the array as an empty array is also a truthy value